samples/graph-pinned-messages/nodejs/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "grapgpinnedmessage-nodejs",
+  "version": "1.0.0",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start": "node server.js"
+  },
+  "author": "",
+  "license": "ISC",
+  "description": "",
+  "dependencies": {
+    "@azure/msal-node": "^1.4.0",
+    "@azure/identity": "^2.1.0",
+    "@microsoft/microsoft-graph-client": "^2.2.1",
+    "@microsoft/teams-js": "^1.10.0",
+    "axios": "^0.21.1",
+    "body-parser": "^1.19.0",
+    "cors": "^2.8.5",
+    "dotenv": "^8.2.0",
+    "ejs": "^3.1.6",
+    "express": "^4.17.1",
+    "isomorphic-fetch": "^3.0.0",
+    "node-fetch": "^2.6.1",
+    "nodemon": "^2.0.7",
+    "parallel-each": "^1.1.7",
```

Comment:
Are we using these packages ?

Reply:
Yes apart from parallel-each.. these packages are required